scipy/special/tests/test_log_softmax.py

```diff
@@ -8,6 +8,14 @@
 
 @pytest.mark.parametrize('x, expected', [
     (np.array([1000, 1]), np.array([0, -999])),
+    # we shouldn't return zero on the smallest subnormal input
+    (np.array([-np.log(np.finfo(np.float32).smallest_subnormal), 0], dtype=np.float32),
+     np.array([float.fromhex('-0x1.00000p-149'), float.fromhex('-0x1.9d1dap+6')],
```

Comment:
This test (I believe) is the one test that fails on Windows with
```
Not equal to tolerance rtol=1e-13, atol=0

Mismatched elements: 1 / 2 (50%)
Max absolute difference among violations: 5.e-324
Max relative difference among violations: 1.
 ACTUAL: array([   0.      , -744.440072])
 DESIRED: array([-4.940656e-324, -7.444401e+002])
```
This seems like a bug in the windows implementation of floating point arithmetic.  What should we do?  Is there a way to relax this test only on Windows?